pkg/bench/rttanalysis/testdata/benchmark_expectations

```diff
@@ -32,21 +32,21 @@ exp,benchmark
 16,CreateRole/create_role_with_3_options
 14,CreateRole/create_role_with_no_options
 16,DropDatabase/drop_database_0_tables
-17,DropDatabase/drop_database_1_table
-18,DropDatabase/drop_database_2_tables
-19,DropDatabase/drop_database_3_tables
+18,DropDatabase/drop_database_1_table
+20,DropDatabase/drop_database_2_tables
+22,DropDatabase/drop_database_3_tables
```

Comment:
I'm okay with this for now, but this is going to hurt in the future. I think ideally we'll find a way to pre-fetch all of the descriptors and zones in a batch. Not for this PR